### PR TITLE
fix(utm): add utm params to console links

### PR DESCRIPTION
**Pull request by jharrell**
Resolves WEB-276

### CODE REVIEW

File: tsconfig.json [hunk 1]
```diff
@@ -4,6 +4,7 @@
   "exclude": ["build"],
   "compilerOptions": {
     "baseUrl": ".",
-    "types": ["docusaurus-plugin-sass"]
+    "types": ["docusaurus-plugin-sass"],
+    "strictNullChecks": true
```
Comment (jharrell) [inline tsconfig.json:8]: important change 1: https://www.typescriptlang.org/tsconfig/#strictNullChecks

File: src/theme/NavbarItem/NavbarNavLink.tsx [hunk 2]
```diff
@@ -38,6 +39,22 @@ export default function NavbarNavLink({
         ),
       };
 
+  if (href?.includes("console.prisma.io")) {
+    return (
+      <BrowserOnly>
```
Comment (jharrell) [inline src/theme/NavbarItem/NavbarNavLink.tsx:44]: important change 2: when rendering any navbar link that directs to `console.prisma.io` check query params.

File: src/theme/MDXComponents.tsx [hunk 3]
```diff
@@ -87,6 +88,33 @@ const ParallelBlocks: React.FC<React.PropsWithChildren> = ({ children }) => {
   );
 };
 
+const DocsLink: React.FC<React.PropsWithChildren<ComponentProps<typeof Link>>> = ({
```
Comment (jharrell) [inline src/theme/MDXComponents.tsx:93]: important change 3: wrap all links in the docs. If the link is to `console.prisma.io` check query params and add them.

**Comment by mhessdev** [comment]
This looks good; I'm just a tad bit anxious about it affecting all links on docs. I assume our link checkers will check if this breaks anything? Are we able to 100% confirm links will not be broken from this?

**Review by meletj — APPROVED** [review]

**Pull request merged**